avogadro/CMakeLists.txt

```diff
@@ -50,12 +50,17 @@ if(USE_QT)
     add_subdirectory(molequeue)
     include_directories(${CMAKE_CURRENT_BINARY_DIR}/molequeue)
   endif()
+
+  if(USE_VTK)
+    include_directories(${CMAKE_CURRENT_BINARY_DIR}/vtk)
```

Comment:
We should move the add_subdirectory(vtk) up to this block too, and get rid of the block below. The VTK we are building requires Qt too, and so that should be represented in the logic. Other than that looks good.